### ISSUE TITLE
MeanCloud__ForgesHold1M: Random iseue reports for chunks and PvP play

### ISSUE THREAD

**Issue by doughphunghus** [doughphunghus/MeanCloud_7DtD#8]
as reported in: https://7daystodiemods.com/forges-hold-1m/?unapproved=3935&moderation-hash=129ef7bd6901a8e748de54ca5ee936ca#comment-3935

- " pretty sure this mod corrupted my chunk, known bug in A18 and A19 apparently when the Forge is overworked"
- "Whenever I put something in the forge, my teammate receives error messages and can no longer play. Mod doesn’t work."

**Comment by doughphunghus** [comment]
EDIT: I just tested MeanCloud__ForgesHold1M mod on a19.4 b7 (as the only mod loaded) and was unable to reproduce this issue. Forge accessible while smelting iron/clay/etc, no console errors, etc.  I did not try to smelt 1,000,000 items with a secondary mod, just noting "basic mod" appears functional.  The last major issue with this mod was because another person loaded another forge mod and they did not play together.

Leave open, as it may be an issue with total amount of resources exceeding some built in maximum number.
